### ISSUE TITLE
request: jfrog rt ls

### ISSUE THREAD

**Issue by gsemet** [jfrog/jfrog-cli#198]
I would love being able to list files or directories from a artifactory path, ignoring subfolder.

Example:
```bash
$ jfrog rt ls repo/path/to/foder
```
and it would list the content of subfolder.

`rt search`
only displays files, and does not display folder.

Currenytly, it seems it is only feasible through [spec file](https://stackoverflow.com/questions/48752133/how-to-list-subfolders-in-artifactory)

Thanks

**Comment by eyalbe4** [comment]
@gsemet - 
@yahavi created https://github.com/jfrog/jfrog-cli/pull/413 to support this functionality,

**Comment by yahavi** [comment]
@gsemet 
In addition to the search command, I'd like to inform you that this functionality is also available by the [rt-fs](https://github.com/jfrog/jfrog-cli-plugins/tree/main/rt-fs) JFrog CLI plugin. The `rt-fs` plugin allows you to run "file-system" like commands with Artiactory servers (Supported by JFrog CLI 1.41.0 and above).
You can install the plugin by running:
```
jfrog plugin install rt-fs
```
And then:
```
jfrog rt-fs ls repo/path/to/folder
```

**Comment by eyalbe4** [comment]
Adding to @yahavi's message, the source code of the **rt-fs** plugin is available [here](https://github.com/jfrog/jfrog-cli-plugins/tree/main/rt-fs). 
Big LIKE for this plugin @yahavi!
Since it looks like this requirement has been implemented by both #413 and [rt-fs](https://github.com/jfrog/jfrog-cli-plugins/tree/main/rt-fs), I guess this issue can be closed.
